examples/colors_rgb.rs

```diff
@@ -101,9 +97,9 @@ struct ColorsWidget {
 
 fn main() -> Result<()> {
     install_error_hooks()?;
-    let terminal = init_terminal()?;
+    let backend = CrosstermBackend::stdout()?;
+    let terminal = Terminal::new(backend)?;
     App::default().run(terminal)?;
```

Comment:
Haven't looked into it, just as an idea: how about merging Terminal and Backend? Putting them into a single line is unfriendly to newcomers but they have a similar goal. Terminal abstracts common things for all the backends. Can we put them on the Backends directly by a trait or something like that? Then something like this might be possible in the future:

```suggestion
    let backend = CrosstermBackend::stdout()?;
    App::default().run(backend)?;
```

Will be breaking, might simplify the usage even further and would be more friendly to beginners. If it seems interesting, get another PR/Issue for this to discuss on this?

Reply:
https://github.com/ratatui-org/ratatui/issues/1251